src/CloseButton.js

```diff
@@ -10,12 +11,21 @@ const defaultProps = {
   label: 'Close',
 };
 
-const CloseButton = React.forwardRef(({ label, onClick }, ref) => (
-  <button ref={ref} type="button" className="close" onClick={onClick}>
-    <span aria-hidden="true">&times;</span>
-    <span className="sr-only">{label}</span>
-  </button>
-));
+const CloseButton = React.forwardRef(
+  ({ label, onClick, className, ...props }, ref) => (
+    <button
+      ref={ref}
+      type="button"
+      className={classNames('close', className)}
+      onClick={onClick}
+      aria-label={label}
```

Comment:
what was the rationale for adding aria-label here, theoretically the label below in the button would be read by a screen reader.

Reply:
I just tried to implement the exact specification from the Bootstrap 4 docs: https://getbootstrap.com/docs/4.3/components/toasts/#basic

Reply:
ah so notice the upstream doesn't also do the `sr-only` with hte label in the body of the button, which should be the same thing as aria-label in this case.

```js
<button type="button" class="ml-2 mb-1 close" data-dismiss="toast" aria-label="Close">
 <span aria-hidden="true">&times;</span>
</button>
```